code/game/striketeams/nanotrasen_deathsquad.dm

```diff
@@ -12,15 +12,16 @@
 
 /datum/striketeam/deathsquad/create_commando(obj/spawn_location, leader_selected = 0)
 	var/mob/living/carbon/human/new_commando = new(spawn_location.loc)
-	var/commando_leader_rank = "Major"
+	var/commando_leader_rank = pick("Major", "Rescue Team Leader", "Commander")
```

Comment:
Rescue Team Leader is pretty long for a title. 26 character is MAX_NAME_LEN and this takes up 18 letters.

Also seems a bit off for a deathsquad title.

Reply:
Not sure what to shorten it to but it's intentionally misleading

Reply:
Rescue leader?

Reply:
Sure